Sources/Everglow.Function/VFX/VFXManager.cs

```diff
@@ -10,16 +10,16 @@ namespace Everglow.Commons.VFX;
 
 public class VFXManager : IVFXManager
 {
-	public static readonly CodeLayer[] drawLayers = new CodeLayer[]
-	{
+	public static readonly CodeLayer[] drawLayers =
+	[
 		CodeLayer.PreDrawFilter,
 		CodeLayer.PostDrawProjectiles,
 		CodeLayer.PostDrawTiles,
 		CodeLayer.PostDrawDusts,
 		CodeLayer.PostDrawBG,
 		CodeLayer.PostDrawPlayers,
 		CodeLayer.PostDrawNPCs,
-	};
+	];
```

Comment:
Error CS1525: Invalid expression term '['	in Everglow.Function	Line 13
Error CS0443: Syntax error; value expected	in Everglow.Function	Line 21

Reply:
Unexpectedly, it needs dotnet7(or higher?) to support C#12
I already change the Lang Version into C#11